### ISSUE TITLE
Not getting the *Scale* button in Service tab

### ISSUE THREAD

**Issue by kCyborg** [OpenNebula/one#6249]
**Description**
While deploying/using Services, I'm not able to see the green Scale button in Sunstone webUI interface.

**To Reproduce**
1. Instantiate a new Service.
2. Head to the Roles tab under the newly created service

![image](https://github.com/OpenNebula/one/assets/37458643/09d937f7-2bd1-49ed-bf3d-1f41f8d9ab2a)

3. The expected green Scale button doesn't show up

> Additional info: I noticed the button not showing up while using the [OneKE](https://docs.opennebula.io/6.6/marketplace/appliances/oneke.html) Service, but also tried creating a mock Service and the button doesn't appear :-(

**Expected behavior**
The expected behavior is to see and interact with the green Scale button in the Service type deployment, under the Roles tab, like in the next picture: 

![image](https://github.com/OpenNebula/one/assets/37458643/7d63ecf0-51b8-4920-87c7-1d2c49645264)

**Details**
 - Affected Component: Sunstone
 - Hypervisor: KVM
 - Version: 6.6.2

**Additional context**
I have looked around and there are some occurrences of the Scale word in the following files:
* /etc/one/sunstone-views/mixed/admin.yaml
* /etc/one/sunstone-views/mixed/advanced.yaml
* /etc/one/sunstone-views/mixed/group_admin.yaml

On those files, under I found the `Role.scale` value, which are all set to true:

```yaml
  oneflow-services-tab:
     # [....]
    panel_tabs_actions:
      service_roles_tab:
        Role.scale: true
```

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by kCyborg** [comment]
Hey guys, any kind of help here will be appreciate it

**Comment by tinova** [comment]
are you using the Enterprise or the Community edition?

**Comment by kCyborg** [comment]
Hi there @tinova, I'm currently using OpenNebula EentrepriseEdition v6.6.2

**Comment by tinova** [comment]
In that case I kindly request you to ask in the [OpenNebula Enterprise Support](https://support.opennebula.pro/) portal.

**Comment by Franco-Sparrow** [comment]
> In that case I kindly request you to ask in the [OpenNebula Enterprise Support](https://support.opennebula.pro/) portal.

Hi Mr. @tinova Sorry, we did not understand your last words. My friend @kCyborg is just reporting a possible *bug*. We have been testing the last months on ON EE `6.6.0`, `6.6.1` and now` 6.6.2` and helped to report *bugs* related with *backups* and *monitoring*. Do we need to report ON EE *bugs* in [OpenNebula Enterprise Support](https://support.opennebula.pro/)](https://support.opennebula.pro/hc/en-us) or can be done here on github as we have been doing so far?

Thanks for the hard work from you and your team :)

**Comment by kCyborg** [comment]
Hi there @tinova my enterprise OpenNebula edition doesn't come with any kind of support, just access to private repository. 

We have been providing feedback in the lasts months as we have a dedicated physical laboratory to test new ON versions (a really awesome job btw!!!).

This is a bug in the Community Edition, as Services is part of the Community Edition (please correct me if I'm wrong) not in the Enterprise Edition.

I'm reporting a bug and I'm able to provide any kind of logs, as my team (@Franco-Sparrow and Ignacio) and myself have been doing in the lasts times. Can you please re-open this issue or tag someone from your team able to lend a hand over here?

**Comment by tinova** [comment]
If you don't have access to the Enterprise Support portal this is indeed the right medium to report issues and bugs.

This is actually a known issue, that cannot be solved in OpenNebula 6.6 series automatically since we don't change configuration files. This will be automatically solved in OpenNebula 6.8.

Meanwhile you can solve it manually (and easily :) ) following this comment: https://github.com/OpenNebula/one/issues/6164#issuecomment-1563098696

**Comment by kCyborg** [comment]
> This will be automatically solved in OpenNebula 6.8.

Just out of curiosity, you guys don't use odd numbers to name the versions?



> Meanwhile you can solve it manually (and easily :) ) following this comment: [#6164 (comment)](https://github.com/OpenNebula/one/issues/6164#issuecomment-1563098696)

Thanks you so much, this indeed solved my problem and now I'm able to scale my Service's VMs
![image](https://github.com/OpenNebula/one/assets/37458643/1c5db074-126a-4190-9267-dc63b2d97000)

**Comment by tinova** [comment]
> Just out of curiosity, you guys don't use odd numbers to name the versions?

odd numbers are for development versions
